Spring-Boot/Spring-Boot-Student/src/test/java/tw/waterball/judgegirl/springboot/student/controllers/it/GroupControllerIT.java

```diff
@@ -109,7 +125,66 @@ public void WhenDeleteGroupByNonExistingGroupId_ShouldRespondNotFound() throws E
     }
 
     private ResultActions deleteGroupById(Integer id) throws Exception {
-        return mockMvc.perform(delete(BASE_PATH + "/{groupId}", id));
+        return mockMvc.perform(delete(GROUP_PATH + "/{groupId}", id));
+    }
+
+    @Test
+    public void GiveOneGroupCreated_WhenAddTwoStudentsIntoTheGroup_ShouldAddSuccessfully() throws Exception {
+        GroupView body = createGroupAndGet(GROUP_NAME);
+        StudentView studentA = signUpAndGetStudent("A");
+        StudentView studentB = signUpAndGetStudent("B");
+        int groupId = body.id;
+        addStudentIntoGroup(groupId, studentA.id);
+        addStudentIntoGroup(groupId, studentB.id);
+        Group group = groupRepository.findGroupById(groupId).orElseThrow(NotFoundException::new);
+        assertEquals(2, group.getStudents().size());
+    }
+
+    @Test
+    public void GiveOneStudentIntoCreatedGroup_WhenDeleteOneStudentFromTheGroup_ShouldDeleteSuccessfully() throws Exception {
```

Comment:
`GivenTwoStudentsAddedIntoCreatedGroup_WhenDeleteOneStudentFromTheGroup_ThenGroupShouldHaveOneStudent`

also, strictify the testcase:
add two students, delete one, assert there is one left in the group.